go-controller/pkg/config/config_test.go

```diff
@@ -233,6 +233,9 @@ enable-persistent-ips=false
 [clustermanager]
 v4-transit-switch-subnet=100.89.0.0/16
 v6-transit-switch-subnet=fd98::/64
+
+[userdefinednetworks]
+max-networks=40000
```

Comment:
hmm even with 40,000 test is testing for defaults?
I don't see us using this anywhere